src/redux/actionTypes/index.js

```diff
@@ -1,12 +1,8 @@
-import _ from 'lodash';
-
-export default _.mapKeys([
-  'APP_GET_LOGIN',
-  'APP_GET_API_KEYS',
-  'APP_GET_FORGOT_PASSWORD',
-  'SETTINGS_GET_USER_SETTINGS',
-  'SETTINGS_REMOVE_USER_DEVICE',
-  'SETTINGS_ADD_USER_DEVICE',
-  'APP_GET_CHANGE_PASSWORD',
-  'APP_GET_SIGNUP',
-]);
+export const APP_GET_LOGIN = 'APP_GET_LOGIN';
```

Comment:
I think the old way of doing it was better. We will have to modify the names twice for every change we make. Chances of introducing an accidental bug would be high in this case. @akshatnitd What do you think?